plugins/mcreator-core/procedures/entity_set_oxygen.json

```diff
@@ -0,0 +1,29 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    },
+    {
+      "type": "input_value",
+      "name": "air",
+      "check": "Number"
+    }
+  ],
+  "inputsInline": true,
+  "previousStatement": null,
+  "nextStatement": null,
+  "colour": 175,
+  "mcreator": {
+    "toolbox_id": "playermanagement",
```

Comment:
I think air works for all entities. Only procedures that only work with players should be in playemenagement, other in entity management or data